src/SearchBar.js

```diff
@@ -38,21 +46,40 @@ export function SearchBar(props){
 
   function onSubmit(event){
     event.preventDefault();
-    props.updateRange(rangeInput);
+    props.updateRange(rangeInput);  // frm: calls setCurrRange() in App.js triggering a Mobilize API call and  a re-render
     setZip(input);
+    // ??? frm: Should I add a call to setEventKind() here?
   }
 
   function setRange(input){
-    setRangeInput(input);
-    props.updateRange(input);
+    setRangeInput(input);       // frm: updates local global state
+    props.updateRange(input);   // frm: calls setCurrRange() in App.js triggering a Mobilize API call and a re-render
+  }
+
+  function setEventKind(input) {
+    setEventKindInput(input);       // frm: update local global
+    props.updateEventKind(input);   // frm: update App.js global - triggering re-render of list of events
   }
 
   function setZip(input) {
-    setInput(input);
-    props.updateZip(input);
+    setInput(input);            // frm: updates local state
+    props.updateZip(input);     // frm: calls setCurrZip() in App.js - triggering a Mobilize API call and a re-render
     History.push(window.location.pathname+'?zip='+input);
   }
 
+  /* ??? frm: TODO:
+   *
+   * Someone should help me think about what the right options are below for 
+   * the kinds of events a user can filter on.  I have already removed some
+   * of the kinds of events.  
```

Comment:
Why kinds of events did you remove? Probably worth making some reasonable choices and documenting them

Reply:
I left a rather long comment in the code that provided a URL to the Mobile API where I got the list of possible events as well as the subset of that list that the code presents.  I did not change the list, but I made it clear that others should feel free to do so, and it is pretty obvious how to do so.